package.json

```diff
@@ -13,7 +13,7 @@
     "copy:files": "copyfiles -u 1 ./src/config.json ./build/ && npm run copy:doxy",
     "copy:doxy": "copyfiles -u 2 src/doxy/* build/doxy/",
     "test": "npm run test:mocked && npm run test:server",
-    "test:server": "jest --silent --runInBand --config=jest.server.config.js",
```

Comment:
I think that this has been done by mistake.

Reply:
Yes, indeed it's by mistake.